src/libgpu/src/vulkan/vulkan_staging.cpp

```diff
@@ -28,10 +30,13 @@ Driver::allocTempBuffer(uint32_t size)
    VmaAllocationCreateInfo allocDesc = {};
    allocDesc.usage = VMA_MEMORY_USAGE_CPU_TO_GPU;
 
+
    VkBuffer buffer;
    VmaAllocation allocation;
+   static_assert(sizeof(vk::BufferCreateInfo) == sizeof(VkBufferCreateInfo),
```

Comment:
This is guaranteed by the Vulkan-Hpp spec already.

Reply:
Indeed, you are right. I even see in Vulkan-Hpp


  static_assert( sizeof( BufferCreateInfo ) == sizeof( VkBufferCreateInfo ), "struct and wrapper have different size!" );

So, this is redundant, and all structs are already checked.

Reply:
Removed.